gateway/src/connectors/defira/defira.ts

```diff
@@ -197,10 +181,8 @@ export class Defira implements Uniswapish {
     amount: BigNumber,
     allowedSlippage?: string
   ): Promise<ExpectedTrade> {
-    const nativeTokenAmount: UniswapTokenAmount = new UniswapTokenAmount(
-      baseToken,
-      amount.toString()
-    );
+    const nativeTokenAmount = UniswapTokenAmount.fromRawAmount(baseToken, amount.toString());
```

Comment:
Invalid type for the fetcher.

Reply:
Oh no I just got confused because `UniswapTokenAmount` makes no sense. Fix the naming.